utilities/tools/AzureDevOps/src/devOpsPipelineTemplate.yml

```diff
@@ -41,12 +41,12 @@ stages:
   - stage: Validation
     displayName: Pester tests
     jobs:
-      - template: /.azuredevops/pipelineTemplates/module.jobs.validate.yml
+      - template: /.azuredevops/pipelineTemplates/jobs.validateModulePester.yml
```

Comment:
Is this template used or intended just as a reference?

Reply:
It's a template for a tool that I wrote back in the day to generate the ADO pipelines from the GitHub counterpart. Thinking of it we may want to question whether we want to keep it or not. What do you think?

Reply:
I guess it's more to maintain and now we have 'permanent' ADO pipelines

Reply:
I'd almost be tempted to remove the logic as part of this PR

Reply:
If not used I'd opt for removing it

Reply:
Removed